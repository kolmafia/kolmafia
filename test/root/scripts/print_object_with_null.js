const {print} = require("kolmafia");

try {
  print({'key': null});
} catch (e) {
  print("exception: " + e.message);
}
