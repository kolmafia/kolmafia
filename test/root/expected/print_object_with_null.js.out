exception: Null / undefined values in JS objects cannot be converted to ASH.
